src/JsonApiModel.php

```diff
@@ -32,6 +32,9 @@ protected function getDateAttribute(string $attribute): ?\DateTime
     protected function getRelationSingle(string $relationShipName, string $modelClassName): ?JsonApiModel
     {
         $relation = $this->dataItem->getRelations()[$relationShipName];
+        if (! isset($relation)) {
```

Comment:
Da in 34 ein Array Access steht, sollte das eigentlich ein Warning geben. Wie wäre es mit 
```php
$relation = $this->dataItem->getRelations()[$relationShipName] ?? null;
```

Reply:
ist ergänzt